import/controls/TextSelectionHandle.qml

```diff
@@ -53,12 +54,15 @@ Image {
         } else {
             contentItem.sendAsyncMessage("Browser:SelectionMoveEnd", selectionController.getMarkerBaseMessage(markerTag))
             moving = false
+            rect = Qt.rect(x - Theme.paddingLarge, y, width + 2 * Theme.paddingLarge, height + Theme.paddingLarge)
```

Comment:
```suggestion
            rect = Qt.rect(mouseArea.x, mouseArea.y, mouseArea.width, mouseArea.height)
```

Reply:
That was my first approach but it doesn't work, mouseArea only changes later and even binding to it doesn't seem to work.

Reply:
My point is that now you have style / behaviour related values scattered here and there. 

Given no animators are used even following would be a bit cleaner:
```
    onXChanged: {
        touchArea = Qt.rect(x - Theme.paddingLarge, y, width + 2 * Theme.paddingLarge, height + Theme.paddingLarge)
    }
    onYChanged: {
        touchArea = Qt.rect(x - Theme.paddingLarge, y, width + 2 * Theme.paddingLarge, height + Theme.paddingLarge)
    }
```
Then you'd not need to care about animations and only special case would be visible false and clearing touchArea there.

Reply:
Ok, changed to that. A bit more stress for the cpu doesn't seem to affect performance.

Reply:
Yeap, that's what I checked as well.